disco/util/token.py

```diff
@@ -1,6 +1,6 @@
 import re
 
-TOKEN_RE = re.compile(r'M\w{23}\.[\w-]{6}\..{27}')
+TOKEN_RE = re.compile(r'[M-N]\w{23}\.[\w-]{6}\..{27}')
```

Comment:
I think simply using `\w{24}\.[\w-]{6}\..{27}` provides enough validation for Discord's token format and adds more value to this update.

Who knows when the next character will pop up, and Disco shouldn't have to be updated when this inevitably occurs.